lib/pages/dashboard/dashboard_club_teams.dart

```diff
@@ -27,8 +29,7 @@ class _DashboardClubTeamsState extends State<DashboardClubTeams> with SingleTick
   Repository _repository;
   List<Team> _teams = [];
   int _currentIndex = 0;
-  String _highlightedTeamCode;
-  List<ClassificationSynthesis> _competitionsClassificationSynthesis = [];
+  List<TeamClassificationBloc> classificationBlocs;
```

Comment:
Did you try to use `TeamClassificationBloc` inside `TeamCard` instead of managing a list of blocs in this class ? The pain point with this proposition is you have to send the load event (in `TeamCard`) only when the `active` flag is true to avoid sending HTTP request for cards which are not displayed !
If you want to try, do another commit (don't squash your commits), so you will be able to revert it easily.

Reply:
Oh ok ! I didn't understand what was the active field for ! I'll try it.